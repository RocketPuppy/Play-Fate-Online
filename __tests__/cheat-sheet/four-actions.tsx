import * as React from "react";
import * as renderer from "react-test-renderer";
import FourActions from "../../src/cheat-sheet/four-actions";

test("renders all the pieces", () => {
  const render = renderer.create(<FourActions />);

  expect(render.toJSON()).toMatchSnapshot();
});
