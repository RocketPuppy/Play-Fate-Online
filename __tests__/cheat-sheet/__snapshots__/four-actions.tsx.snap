// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Four Actions
  </h2>
  <dl>
    <dt>
      Overcome
    </dt>
    <dd>
      get past an obstacle
    </dd>
    <dt>
      Create an Advantage
    </dt>
    <dd>
      invoke an aspect for free
    </dd>
    <dt>
      Attack
    </dt>
    <dd>
      harm another character
    </dd>
    <dt>
      Defend
    </dt>
    <dd>
      prevent attacks or advantages on you
    </dd>
  </dl>
</div>
`;
